### DISCUSSION TITLE
"Little Arc" window does not seem to be properly supported

### DISCUSSION THREAD

**Discussion by macintacos** [rxhanson/RectanglePro-Community#281] [General]
In the Arc browser, there is the concept of a window called "Little Arc" that comes up when you click a link from outside of the browser. I find this to be quite handy, so I leave it enabled. However, I just started using Rectangle for the first time in a while, and it seems like the resizing shortcuts/commands do not work properly. For example, the Maximize command just simply doesn't work at all, split left/right _move_ the window, but they do not resize it, etc.

Is this something that Rectangle can resolve, or is this something that I need to reach out to the Arc developers about?

**Comment by rxhanson** [comment]
Thanks for trying Rectangle. This would have to be resolved on the Arc side. Rectangle/Pro will simply tell macOS the desired size and position of a window via the Accessibility API, and it's up to the application's window to respond properly to that request. In Arc's scenario, it's likely that there is some kind of limitation placed on the window size.

**Reply by macintacos** [reply]
Interesting. I can resize the window pretty freely, so maybe they're doing something else to mask the size. I remember it resizing okay on Amethyst, now that I think about it, so maybe this is a new development...

I'll reach out to them to see what they say. Thanks for responding!
